bin/ingest.py

```diff
@@ -242,20 +244,24 @@ def main(args):
             if not os.path.exists(attachments_dest_dir):
                 os.makedirs(attachments_dest_dir)
             for attachment_file in os.listdir(os.path.join(root_path, "additional")):
-                match = re.match(rf"{abbrev}(\d+)_(\w+)\.(\w+)")
-                if match is not None:
-                    paper_num, type_, ext = match.groups()
-                    paper_num = int(paper_num)
-
-                    file_name = f"{collection_id}-{volume_name}.{paper_num}.{type_}.{ext}"
-                    dest_path = os.path.join(attachments_dest_dir, file_name)
-                    log(f"Copying {attachment_file} -> {dest_path}", args.dry_run)
-                    if not args.dry_run:
-                        shutil.copyfile(attachment_file, dest_path)
+                attachment_file_path = os.path.join(root_path, "additional", attachment_file)
+                match = re.match(rf"{year}\.{venue_name}-\w+\.(\d+)_?(\w+)\.(\w+)$", attachment_file)
+                if match is None:
+                    print(f"* Warning: no attachment match for {attachment_file}", file=sys.stderr)
+                    sys.exit(2)
+
+                paper_num, type_, ext = match.groups()
+                paper_num = int(paper_num)
+
+                file_name = f"{collection_id}-{volume_name}.{paper_num}.{type_}.{ext}"
+                dest_path = os.path.join(attachments_dest_dir, file_name)
+                # log(f"Copying {attachment_file} -> {dest_path}", args.dry_run)
+                if not args.dry_run:
+                    shutil.copyfile(attachment_file_path, dest_path)
 
-                    collections[collection_id][volume_name][paper_num][
-                        "attachments"
-                    ].append(dest_path)
+                collections[collection_id][volume_name][paper_num][
+                    "attachments"
+                ].append((dest_path, type_))
 
```

Comment:
This is just a bugfix with attachment handling. This is the first time we've had attachments since I made some other updates to the code and I tend to update `ingest.py` lazily.